packages/create-mc-app/src/validations.ts

```diff
@@ -81,10 +79,22 @@ const throwIfNodeVersionIsNotSupported = (
   }
 };
 
+const throwIfPackageManagerVersionIsMissing = (
+  packageManager?: TPackageManager,
+  packageManagerVersion?: string
+) => {
+  if (packageManager && !packageManagerVersion) {
```

Comment:
**nit**: I was wondering why users need to specify the package manager version.
As a developer using the CLI, I can see it asking me about the package manager I want to use but not sure why it would ask me about the version. Most of the time, I wouldn't even know and need to search for it.

Reply:
The version is needed for the `packageManager` field in `package.json`.
Unfortunately it doesn't work without it.

One option I was thinking about was to automatically fetch the latest version.

I was also thinking of automatically detect the package manager but that's tricky because yarn or pnpm don't need to be installed globally.

As a follow up I also wanted to look into enabling yarn or pnpm (if selected) via Corepack. That requires though to have certain nodejs versions installed.

So there are some tricky scenarios to consider and for the sake of simplicity I opted to require the package manager version to be explicitly requested.

Let me take another look at this though, because for pnpm (due to how it should be installed via Corepack) I opted to skip the installation step. So the version is not strictly needed anymore even though the user will need to specify it anyway in the `packageManager` field.

Reply:
Decided to remove it for now: https://github.com/commercetools/merchant-center-application-kit/pull/3087/commits/e2dc2b24c5f314ed79767166cfeac71f0c95c004